packages/components/src/components/chip/chip.tsx

```diff
@@ -8,35 +8,46 @@ import { JSX } from '@stencil/core/internal';
 })
 export class Chip implements ComponentInterface {
 
-  /**
-   * The icon source.
-   *
-   * @type {string}
-   */
-  @Prop() icon?: string;
-
-  /**
-   * The chip content that is displayed in the component.
-   *
-   * @type {string}
-   */
-
-  @Prop() content: string;
-
-  /**
-   * Checks if the chip is active and applies different styling to the component.
-   *
-   * @type {boolean}
-   */
-
-  @Prop() active: boolean = false;
-
-  render(): JSX.Element {
-      return (
-          <div class={`chip ${this.active ? 'chip--active' : ''}`}>
-              {this.icon && <img class="chip__icon" src={this.icon}></img>}
-              <div class="chip__content">{this.content}</div>
-          </div>
-      );
-  }
+    /**
+     * The icon source.
+     *
+     * @type {string}
+     */
+    @Prop() icon?: string;
+
+    /**
+     * The chip content that is displayed in the component.
+     *
+     * @type {string}
+     */
+
+    @Prop() content: string;
+
+    /**
+     * The background color of the chip.
+     * The default HEX value refers to the --brand-colors-dark-pine-100 from MIDT
```

Comment:
Write the hex value as well, so customers don't have to look up in MIDT, which they may not know about.

Reply:
✅